File: 1968-array-with-elements-not-equal-to-average-of-neighbors/1968-array-with-elements-not-equal-to-average-of-neighbors.java
class Solution {
    public int [] rearrangeArray(int[] nums) {
		Arrays.sort(nums);
		int [] ans = new int[nums.length];
		
		int r = nums.length - 1, l = 0, indx = 0;
		
		while(l < r)
		{
			ans[indx] = nums[l];
			indx++;
			ans[indx] = nums[r];
			l++;
			r--;
			indx++;
		}
		
		if (nums.length%2 != 0)
            ans[indx] = nums[l];
		
		return ans;
	}
}

File: 1968-array-with-elements-not-equal-to-average-of-neighbors/1968-array-with-elements-not-equal-to-average-of-neighbors.py
class Solution:
    def rearrangeArray(self, nums: List[int]) -> List[int]:
        nums.sort()
        ans = []
        l, r = 0, len(nums) - 1
        while l <= r:
            ans.append(nums[l])
            if l != r:
                ans.append(nums[r])
            l += 1
            r -= 1
        return ans